datalad/tests/test_nonasyncrunner.py

```diff
@@ -15,14 +15,29 @@
 import subprocess
 from time import sleep
 
-from datalad.tests.utils import assert_false, assert_true, eq_, \
-    known_failure_windows, known_failure_osx, with_tempfile
+from datalad.tests.utils import (
+    SkipTest,
+    assert_false,
+    assert_true,
+    eq_,
+    known_failure_windows,
+    known_failure_osx,
+    with_tempfile,
+)
+
+from datalad.cmd import (
+    StdOutCapture,
+    WitlessProtocol,
+    WitlessRunner,
+)
+from datalad.nonasyncrunner import (
+    _ReaderThread,
+    run_command,
+)
+
+from datalad.utils import on_windows
```

Comment:
at some point we should really agree on whether we use relative or absolute imports, and whether exclusively or "mixed" (e.g. absolute generally, and relative within tests of direct imports of "one level upstairs" thus making it easy to move needed code without requiring to rewrite all the imports)

Reply:
> thus making it easy to move needed code without requiring to rewrite all the imports

You use an IDE - couldn't be easier ;-)

Reply:
Well, might then with teach git to recognize IDE moved code as a "move" then too (file content gets changed, not an exact/clean move). Use of absolute import for such use cases provides no advantage whatsoever AFAIK, or what is it?

Reply:
I think we agreed, and it was absolute and not mixed. I am working towards this goal for a long time. I change every code that I touch.

re code moving: The moving that I am doing is mostly between packages, and relative imports are not helping with that.